src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -122,14 +122,14 @@ protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
 		for ( final String attribute : AttributesProvider.getInstance().getAttributes() ) {
 			final String attributeValue = webData.get( attribute );
-			if ( attributeValue != null && !isDefault( attributeValue ) ) {
+			if ( attributeValue != null && !isDefault( attribute, attributeValue ) ) {
 				state.put( attribute, attributeValue );
 			}
 		}
 		return state;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=32&issues=AWSLWuHMsk71k7lwmQ83&open=AWSLWuHMsk71k7lwmQ83)

[ISSUE_KEY]: AWSLWuHMsk71k7lwmQ83